index.js

```diff
@@ -115,7 +115,7 @@ const openArchive = function (file, noCache) {
 // ensure non-breaking changes
 exports.get = (file, archive) => {
   log.debug('get() : ' + file + ', ' + archive);
-  if (['darwin', 'linux'].indexOf(process.platform) > -1) {
+  if (storm && ['darwin', 'linux'].indexOf(process.platform) > -1) {
```

Comment:
As stated in issue #21, storm-replay will not build in node12 due to deprecated calls.   It will need to be updated.  As of right now, storm-replay builds in node v10 and node-gyp v6.1.0.

storm-replay DOES have an issue with node-gyp v7.